.github/workflows/tracegen.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Push tracegen to Github packages
         uses: docker/build-push-action@v3
         with:
-          context: tracegen
+          context: cmd/tracegen
```

Comment:
Do we also need to update `publish-stable` (`docker build tracegen` => `docker build cmd/tracegen`)?